source/Applications/Advanced/HandEyeCalibration/UtilizeEyeInHandCalibration/packages/YamlDotNet.8.1.2/LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2008, 2009, 2010, 2011, 2012, 2013, 2014 Antoine Aubry and contributors
```

Comment:
I am not sure whether we want to host the packages in the repo. SE needs to be involved. Even if we do, this is probably not the right directory.

Reply:
Removing the packages! Instead, Im explaining in the readme how to install them